grails-app/views/_js_includes.gsp

```diff
@@ -55,7 +55,7 @@
     <script type="text/javascript" src="${resource(dir: 'js/portal', file: 'portal.js')}"></script>
     <script type="text/javascript" src="${resource(dir: 'js/portal', file: 'ObjectFactory.js')}"></script>
     <script type="text/javascript" src="${resource(dir: 'js/portal', file: 'PortalEvents.js')}"></script>
-    <script type="text/javascript" src="${resource(dir: 'js/portal', file: 'WKTNormalised.js')}"></script>
+    <script type="text/javascript" src="${resource(dir: 'js/portal/prototypes', file: 'WKTNormalised.js')}"></script>
```

Comment:
Why not? It's overriding an OpenLayers class. Still cleaner than having it on the root of the portal JS directory!! :)